### PR TITLE
[BugFix] Fix numeric overflow when print empty column

### PR DESCRIPTION
**Pull request by liuyehcf**
## What type of PR is this：
- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by liuyehcf** [comment]
run starrocks_be_unittest

**Comment by liuyehcf** [comment]
run starrocks_admit_test

**Comment by liuyehcf** [comment]
run starrocks_be_unittest

**Comment by liuyehcf** [comment]
run starrocks_admit_test

**Comment by liuyehcf** [comment]
run starrocks_admit_test

**Comment by liuyehcf** [comment]
run starrocks_admit_test

**Comment by liuyehcf** [comment]
run starrocks_admit_test

**Comment by liuyehcf** [comment]
run starrocks_admit_test

File: be/src/column/fixed_length_column_base.cpp [hunk 1]
```diff
@@ -290,7 +290,7 @@ template <typename T>
 std::string FixedLengthColumnBase<T>::debug_string() const {
     std::stringstream ss;
     ss << "[";
-    size_t size = this->size();
+    int size = this->size();
```
Comment (kangkaisen) [inline be/src/column/fixed_length_column_base.cpp:293]: I think you should change the for loop int to size_t. not change int to size_t in this line.

Comment (liuyehcf) [reply be/src/column/fixed_length_column_base.cpp:293]: The problem is `size - 1` may overflow if size is type `size_t`, changing `int i` to `size_t i` don't work out

Comment (kangkaisen) [reply be/src/column/fixed_length_column_base.cpp:293]: O，I see

**Review by kangkaisen — APPROVED** [review]

**Review by Seaven — APPROVED** [review]

**Pull request merged**